### PR TITLE
base implementation of homepage

### PR DESCRIPTION
**Pull request by ArturWk4**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "^4.3.1",
```
Comment (screamcha) [inline package.json]: Do you use this dependency in the application?

File: public/index.html [hunk 2]
```diff
@@ -2,42 +2,12 @@
...
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
+    <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (screamcha) [inline public/index.html]: Please remove imported scripts

File: src/components/header/Header.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.logo {
+    background-color: #243d6b;
```
Comment (screamcha) [inline src/components/header/Header.css]: Please put all colors in a separate file

File: src/components/header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Menu from "../menu/Menu.jsx";
+import { Navbar } from "react-bootstrap";
+import "./Header.css";
+import {Container} from 'react-bootstrap'
```
Comment (screamcha) [inline src/components/header/Header.jsx]: Try to apply the following order of imports:
1. imports from external modules (e.g. `import React from "react"`);
2. imports of your modules (e.g. `import Menu from "../menu/Menu.jsx");
3. imports of .css, .scss etc., images, files etc.

File: src/components/top-films/TopFilms.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import "./TopFilms.css";
+import TopFilmsItem from "../top-films-item/TopFilmsItem.jsx";
+import { CardDeck } from "react-bootstrap";
+
+const TopFilms = () => {
+  // setfilms() will be used when I'll load data from server
+  // eslint-disable-next-line no-unused-vars
```
Comment (screamcha) [inline src/components/top-films/TopFilms.jsx]: If you want to disable rules then please do it for the whole project. Lines like that looks like dirt in your code.

**Comment by screamcha** [comment]
I don't see any sass files in your project. Please update your styles.

File: package.json (hunk 1 above)
Comment (ArturWk4) [reply package.json]: Yes, react-bootstrap requires this dependency

File: public/index.html (hunk 2 above)
Comment (ArturWk4) [reply public/index.html]: I just forgot to remove it

File: src/components/header/Header.css (hunk 3 above)
Comment (ArturWk4) [reply src/components/header/Header.css]: ok

File: src/components/header/Header.jsx (hunk 4 above)
Comment (ArturWk4) [reply src/components/header/Header.jsx]: okay

File: src/components/top-films/TopFilms.jsx (hunk 5 above)
Comment (ArturWk4) [reply src/components/top-films/TopFilms.jsx]: pkay

File: src/components/hero-image/HeroImage.scss [hunk 6]
```diff
@@ -9,7 +8,7 @@
 }
 
 .hero h1 {
```
Comment (screamcha) [inline src/components/hero-image/HeroImage.scss]: Please use nesting in sass instead.

Comment (ArturWk4) [reply src/components/hero-image/HeroImage.scss]: ok

**Comment by AndreiShenets** [comment]
@ArturWk4  Please make repository private and make @AndreiShenets  as collaborator

**Pull request merged**